### DISCUSSION TITLE
vercel not support python 3.11 version

### DISCUSSION THREAD

**Discussion by rinshadkv** [vercel/community#2683] [Help] [answered]
### Problem Description

HI folks ,
I am  new to  vercel. I am  try to deploy a Django project on vercel. I am getting error like this 
<html>
<body>

./build_files.sh: line 2: python3.11: command not found
--
22:43:46.415 | ./build_files.sh: line 3: python3.11: command not found

</body>
</html>
please help me solve this problem

**Comment by ryandotelliott** [comment, marked as answer]
Vercel does not support Python 3.9. See: https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/python#python-version
